website/docusaurus.config.esm.mjs

```diff
@@ -88,6 +88,9 @@ const config = createDocusaurusConfig({
             indexName: "goauthentik",
         },
     },
+    customFields: {
+        currentVersion: "2025.4.1",
```

Comment:
can you use the version field of the root package.json file instead of having to have bumpversion in here